tests/test_interfaces.py

```diff
@@ -420,3 +421,65 @@ def test_output_format(
 
     assert processed_result["timepoint_id"].dtype == np.int64
     assert processed_result["sample_id"].dtype == np.int64
+
+
+# Tests to assert that incorrect input to calibrate is caught
+@pytest.mark.parametrize("model_fixture", MODELS)
+def test_positive_int_calibrate(model_fixture):
```

Comment:
@sabinala , could you actually remove this test. It's already covered by the existing `calibrate` tests.

Reply:
Done